saleor/settings.py

```diff
@@ -577,13 +577,16 @@ def get_host():
 ]
 
 # Plugin discovery
+EXTERNAL_PLUGINS = []
 installed_plugins = pkg_resources.iter_entry_points("saleor.plugins")
 for entry_point in installed_plugins:
     plugin_path = "{}.{}".format(entry_point.module_name, entry_point.attrs[0])
-    if plugin_path not in PLUGINS:
+    if plugin_path not in EXTERNAL_PLUGINS:
```

Comment:
If `plugin_path` exists in `BUILTIN_PLUGINS` but not in `EXTERNAL_PLUGINS`, this will include two instances of the same plugin. You should also check if `plugin_path` doesn't exist in  `BUILTIN_PLUGINS`